qiskit/providers/aer/backends/qasm_simulator.py

```diff
@@ -192,6 +192,31 @@ class QasmSimulator(AerBackend):
       OpenMP parallelization. If parallel circuit or shot execution
       is enabled this will only use unallocated CPU cores up to
       max_parallel_threads (Default: 100).
+
+    These backend options only apply when using the ``"matrix_product_state"``
+    simulation method:
+
+    * ``"approximation_type"`` (int): Can be one of the following options:
+      NO_APPROX - means no approximation.
+      ABSOLUTE_APPROX - only the largest max_num_coefficients_for_approx
+      Schmidt coefficients are retained.
+      RELATIVE_APPROX - if the number of Schmidt coefficients is larger than
+      max_num_coefficients_for_approx, then the smallest values are
+      discarded (see approx_thresold).
+      (Default: NONE).
+
+    * ``"max_num_coefficients_for_approx"`` (int): Sets a limit on the number of Schmidt
+      coefficients retained at the end of the svd algorithm.
+      If the number of coefficients is greater than max_num_coefficients_for_approx,
+      small values may be discarded, depending on approximation_type and on approx_threshold.
+      (Default: UINT_64MAX).
+
+    * ``"approx_threshold"`` (double): Sets a lower limit on the value of the Schmidt
```

Comment:
Same comment as for controller: I think this option should just be `"matrix_product_state_bond_truncation_threshold"` and should work independently of the max bond dimension truncation if set